etc/docker/elyra/start-elyra.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+export NB_PORT=${NB_PORT:-8888}
+export GATEWAY_HOST=${GATEWAY_HOST:-localhost}
+export KG_URL=${KG_URL:-http://${GATEWAY_HOST}:${NB_PORT}}
+export KG_HTTP_USER=${KG_HTTP_USER:-jovyan}
```

Comment:
Should the default value here be `${NB_USER}`?  A `docker inspect` of the base image shows it will be in the environment:
```
           "Env": [
                "PATH=/opt/conda/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin",
                "DEBIAN_FRONTEND=noninteractive",
                "CONDA_DIR=/opt/conda",
                "SHELL=/bin/bash",
                "NB_USER=jovyan",
  ```

Reply:
I will probably clean up this section, and remove some of these, but will have this in mind, which is mostly from JEG.

Reply:
Sounds good.  Here's the [EG doc reference](https://jupyter-enterprise-gateway.readthedocs.io/en/latest/getting-started.html#notebook-6-0-and-above).